views/get-started.erb

```diff
@@ -9,112 +9,54 @@
+			</p>
 
-			<h2 class="task-list-section"><span class="task-list-section-number">8.</span> After your beta service assessment</h2>
-			<div class="task-list-items">
-				<ul class="list list-bullet">
-					<li>Change your support plan</li>
-					<li>Request a new service.gov.uk URL and re-configure the CDN</li>
-				</ul>
-			</div>
+			<h3 class="heading-medium">Push your first app</h3>
+			<p>
+				Once we set up your account and you have <a href="https://docs.cloud.service.gov.uk/#setting-up-the-command-line">installed the Cloud Foundry CLI</a>, you can access GOV.UK PaaS, set a password and start using the platform straight away. Our <a href="https://docs.cloud.service.gov.uk/#quick-setup-guide">quick setup guide</a> will help you deploy a static site to test your connection and then <a href="https://docs.cloud.service.gov.uk/#deploying-apps">push some sample code</a>, use the marketplace to <a href="https://docs.cloud.service.gov.uk/#deploy-a-backing-or-routing-service">request backing services like databases</a>, and set up multiple environments and a development pipeline.
+			</p>
 
-			<h2 class="task-list-section"><span class="task-list-section-number">9.</span> Enter the public beta phase with your service</h2>
-			<div class="task-list-items">
-				<ul class="list list-bullet">
-					<li>Continue to iterate your service based on user feedback and production needs, adding other services (e.g. autoscaling) where needed</li>
-					<li>Pass final assessment</li>
-				</ul>
-			</div>
+			<h3 class="heading-medium">Support in trial mode</h3>
+			<p>
+				Check the documentation to <a href="https://docs.cloud.service.gov.uk/#troubleshooting">troubleshoot common problems</a>. If you can’t find a solution our team can help during office hours. Contact us through our <a href="/support">support page</a>.
+			</p>
 
-			<h2 class="task-list-section"><span class="task-list-section-number">10.</span> After live assessment</h2>
-			<div class="task-list-items">
-				<ul class="list list-bullet">
-					<li>Keep your dependencies up to date</li>
-					<li>Be ready to redeploy if there are vulnerabilities in language runtimes</li>
-				</ul>
-			</div>
+			<h3 class="heading-medium">Increase your quota</h3>
 			<p>
-				<a name="asterisk"></a>* These services may be available through the PaaS platform in the future rather than as separate procurements; please see our roadmap for more information.
+				If you decide to increase your quota and make full use of the platform on a paid basis, get in contact with <a href="mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk">gov-uk-paas-support@digital.cabinet-office.gov.uk</a>.<br />
+				Procurement starts with signing a Memorandum of Understanding. You might not need to do this if your department has already signed one.<br />
+				Finance colleagues will help setting up billing arrangements.
 			</p>
 
+			<a class="button" href="/signup">Request a trial account</a>
```

Comment:
@stephenjoe1 Is this spacing between the last paragraph and the button OK?

![screen shot 2017-11-22 at 15 11 37](https://user-images.githubusercontent.com/260438/33134409-9256586e-cf97-11e7-8fdd-d7ddd066c215.png)